packages/apps/plugins/plugin-space/package.json

```diff
@@ -68,6 +69,7 @@
     "vite": "^5.0.12"
   },
   "peerDependencies": {
+    "@babel/core": "^7.0.0-0",
```

Comment:
I'm guessing this is unnecessary too and was needed because of vue? If not, I'd be curious what's causing this to be required

Reply:
That one was called out explicitly, see below.

I guess the others should be added too. Was mostly trying to unblock the agent build, but the lack of the rest would fail the CI check if I ever get that working.

``` ERR_PNPM_PEER_DEP_ISSUES  Unmet peer dependencies

docs
└─┬ @vuepress/plugin-search 2.0.0-rc.14
  └── ✕ unmet peer vuepress@2.0.0-rc.7: found 2.0.0-rc.0

packages/apps/plugins/plugin-space
└─┬ react-drag-drop-files 2.3.10
  └─┬ styled-components 5.3.11
    └─┬ babel-plugin-styled-components 2.1.4
      └─┬ @babel/plugin-syntax-jsx 7.23.3
        └── ✕ missing peer @babel/core@^7.0.0-0
Peer dependencies that should be installed:
  @babel/core@^7.0.0-0

packages/apps/plugins/plugin-stack
└─┬ react-drag-drop-files 2.3.10
  └─┬ styled-components 5.3.11
    └─┬ babel-plugin-styled-components 2.1.4
      └─┬ @babel/plugin-syntax-jsx 7.23.3
        └── ✕ missing peer @babel/core@^7.0.0-0
Peer dependencies that should be installed:
  @babel/core@^7.0.0-0

packages/devtools/devtools
└─┬ react-drag-drop-files 2.3.10
  └─┬ styled-components 5.3.11
    └─┬ babel-plugin-styled-components 2.1.4
      └─┬ @babel/plugin-syntax-jsx 7.23.3
        └── ✕ missing peer @babel/core@^7.0.0-0
Peer dependencies that should be installed:
  @babel/core@^7.0.0-0```

Reply:
https://github.com/dxos/dxos/commit/e1c21d948e3812a2778943a83cb68f9587e640f7 adds the rest if they should be added